### ISSUE TITLE
BC: Errors when selected bubble has no data when switching an indicator

### ISSUE THREAD

**Issue by angieskazka** [Gapminder/tools-page#198]
Make trail for nigeria, switch Y to literacy rate, adult
got 2 popular errors

![image](https://user-images.githubusercontent.com/3648190/113446016-35938e80-93f7-11eb-8364-a0c52fd71dc5.png)

**Comment by jheeffer** [comment]
Probably fixed.
Nigeria does have data for literacy rate so not sure if your example was correct or not specific enough. Do you mean like China, it has not until frame domain limit 2011? Then probably duplicate of https://github.com/Gapminder/tools-page/issues/190

**Comment by angieskazka** [comment]
retested with electricity use per person in afghanistan, all good
